service/src/main/java/uk/gov/hmcts/reform/fpl/service/FurtherEvidenceNotificationService.java

```diff
@@ -58,18 +60,43 @@ public Set<String> getSecondaryLocalAuthorityRecipients(CaseData caseData) {
         return localAuthorityRecipients.getRecipients(recipientsRequest);
     }
 
-    public Set<String> getRepresentativeEmails(CaseData caseData, DocumentUploaderType userType) {
-        List<RepresentativeRole.Type> roles = List.of(CAFCASS, RESPONDENT);
+    public Set<String> getRepresentativeEmails(CaseData caseData) {
         HashSet<String> emails = representativesInbox.getRepresentativeEmailsFilteredByRole(caseData,
-            DIGITAL_SERVICE, roles);
+            DIGITAL_SERVICE, List.of(CAFCASS, RESPONDENT));
```

Comment:
Don't we already get the respondent solicitor emails in the next line? Would this just need to be CAFCASS?